### PR TITLE
Feature/one file pr scenario #000

### PR DESCRIPTION
**Pull request by Lanestolen**

### CODE REVIEW

**Review by rasmim — APPROVED** [review]

File: scripts/clean.sh [hunk 1]
```diff
@@ -18,9 +18,9 @@ done
 
 
 sudo ovs-vsctl del-br $bridge
-sudo ip link del ${bridge}_vlan10 mode tap
-sudo ip link del ${bridge}_vlan20 mode tap
-sudo ip link del ${bridge}_vlan30 mode tap
+sudo ip link del ${bridge}_vlan10 mode
+sudo ip link del ${bridge}_vlan20 mode
+sudo ip link del ${bridge}_vlan30 mode
```
Comment (rasmim) [inline scripts/clean.sh:23]:
```
sudo ip link del ${bridge}_vlan10
sudo ip link del ${bridge}_vlan20
sudo ip link del ${bridge}_vlan30

```

Comment (rasmim) [reply scripts/clean.sh:23]: it is working without the `mode` keyword at the end.

**Review by rasmim — CHANGES_REQUESTED** [review]
clean script. small update

**Comment by rasmim** [comment]
I tried to run it on a development server. I got this error: 

```
7:13PM ERR mirroring traffic error="exit status 1: ovs-vsctl: select_vlan=: argument does not end in \"=\" followed by a value."
7:13PM ERR configuring monitoring error="exit status 1: ovs-vsctl: select_vlan=: argument does not end in \"=\" followed by a value."
```

Can you please investigate @Lanestolen ?

**Comment by Lanestolen** [comment]
Can you send me the scenario file used during this test?

**Comment by rasmim** [comment]
I checked another time, it is no problem with the pull request. I was my mistake, using a scenario number that was not existing. 

`DEFAT_SSL_OFF=true DEFAT_PORT=5554 go run app/client/main.go start -n "dnst" -t "dnst" -s 1`


It can be merged, I will be nice to have a panic if the scenario number does not exist.

**Review by rasmim — APPROVED** [review]

**Review by rasmim — APPROVED** [review]

**Comment by rasmim** [comment]
Please proceed with merge :)

**Pull request merged**